### ISSUE TITLE
[Request] Option to Search System Settings

### ISSUE THREAD

**Issue by colwellkr** [prateekmedia/Menu11#46]
I've been using Menu 11 for a few weeks now, first on Kubuntu and now on Arch; it's really solid and nice to use, so thanks to the devs for all of the hard work.

One request that would make it even more intuitive to use would be the ability to search KCM's system settings. It could be added as a toggle in the Menu 11 settings, and would search through settings/menus in the KConfig Module.

This should be possible as the Application Menu widget is capable of such a search, though I don't know enough myself to implement it.

Please let me know if I can help or provide any clarification!

**Comment by prateekmedia** [comment]
I can search from settings easily, Like Typying display gives me Display Configuration result and typing appearance gives me appearance related results in the menu

![Screenshot_20211117_142343](https://user-images.githubusercontent.com/41370460/142168699-b62674a6-bd0e-4bae-90cc-9baa822f1031.png)

**Comment by colwellkr** [comment]
Strange, perhaps there's a bug or something odd with my system?

I'm Arch with SDDM and Plasma Wayland. Are there any diagnostics I can pull, libraries I might be missing, or anything I can do to help figure out what's going on?

![image](https://user-images.githubusercontent.com/8947323/142209845-6779eb39-877c-4fec-a627-e23f884bfb62.png)

**Comment by prateekmedia** [comment]
Yes you are right, I am on KDE 5.23 (so are you I think) and it has same result as you, looks like it is a problem with 5.23, It worked (as I showed in screenshot) on 5.22

**Comment by colwellkr** [comment]
Ahhh, yep. I'm on KDE Plasma 5.23.1. So if it's working on 5.22 then there must be some breaking change in the interim.

I'll update the title to bug then. Thanks for looking into it; let me know how I can help!
